docs/mvvm/generators/INotifyPropertyChanged.md

```diff
@@ -0,0 +1,38 @@
+---
+title: INotifyPropertyChanged attributes
+author: Sergio0694
+description: Attributes to inject MVVM support into partial classes
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, mvvm, componentmodel, property changed, notification, binding, net core, net standard
+dev_langs:
+  - csharp
+---
+
+# INotifyPropertyChanged attributes
+
+The [`INotifyPropertyChanged`](/dotnet/api/communitytoolkit.mvvm.componentmodel.INotifyPropertyChangedAttribute) type is an attribute that allows inserting MVVM support code into existing types. Along with other related attributes ([`ObservableObject`](/dotnet/api/communitytoolkit.mvvm.componentmodel.ObservableObjectAttribute) and [`ObservableRecipient`](/dotnet/api/communitytoolkit.mvvm.componentmodel.ObservableRecipientdAttribute)), its purpose is to support developers in cases where the same functionality from these types was needed, but the target types were already implementing from another type. Since C# does not allow multiple inheritance, these attributes can instead be used to have the MVVM Toolkit generator add the same code right into those types, sidestepping this limitation.
+
+> [!NOTE]
+> These attributes are only meant to be used in cases where the target types cannot just inherit from the equivalent types (eg. from `ObservableObject`). If that is possible, inheriting is the recommended approach, as it will reduce the binary size by avoiding creating duplicated code into the final assembly.
+
+> **Platform APIs:** [`INotifyPropertyChanged`](/dotnet/api/communitytoolkit.mvvm.componentmodel.INotifyPropertyChangedAttribute), [`ObservableObject`](/dotnet/api/communitytoolkit.mvvm.componentmodel.ObservableObjectAttribute), [`ObservableRecipient`](/dotnet/api/communitytoolkit.mvvm.componentmodel.ObservableRecipientdAttribute)
+
+## How to use them
+
+Using any of these attributes is pretty straightforward: just add them to a [partial class](/dotnet/csharp/programming-guide/classes-and-structs/partial-classes-and-methods) and all the code from the corresponding types will automatically be generated into that type. For instance, consider this:
+
+```csharp
+[INotifyPropertyChanged]
+public partial class MyViewModel : SomeOtherType
+{    
+}
+```
+
+This will generate a complete `INotifyPropertyChanged` implementation into the `MyViewModel` type, complete with additional helpers (such as `SetProperty`) that can be used to reduce verbosity. Here is a brief summary of the various attributes:
```

Comment:
This block is a bit hard to parse visually with all the blocks:

![image](https://user-images.githubusercontent.com/24302614/179041513-8c04a8ef-34bc-4948-94d0-a78dd023913c.png)

At least in certain wrappings. Maybe don't fence all the things or make the starting ones bold, not sure the best way to resolve?

Reply:
I've changed the titles to be links, which also makes them blue so they stand out more 🙂